src/test/java/ToRemoveTest.java

```diff
@@ -0,0 +1,74 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.util.Locale;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ToRemoveTest {
+
+    @Test
+    @DisplayName("testToUpperCase")
+    void toUpperCase() {
+        final String TEST_STRING = "test";
```

Comment:
`TEST_STRING` - that could be correct if that is a class level constant - `private final static ...` . That is not the case here. So - please rename into something like `testString.`